Src/Scripting/NativeDefinitions/PlayerControllerDefinitions.h

```diff
@@ -25,6 +25,7 @@ class PlayerControllerDefinitions : public RefCounter {
 
     public:
         PlayerControllerDefinitions(ScriptManager* mgr, RefCounterManager* rcMgr, Camera* cam);
+        float height;
```

Comment:
This shouldn't be a public member.

Reply:
It is exposed to allow the game to control the camera height.
It should probably be done a better way.
Namely, i'm thinking of how the camera sinks into the ground in the pocket dimension room, and crouching.
No doubt there's a reason the player is a midget, so I didn't want to do too much, but being so tiny was really bugging me.
The desired camera height from the floor seems to be about 25 units.